src/components/layouts/ContentLayout/ContentLayout.module.scss

```diff
@@ -14,7 +14,7 @@
 .pageHeader {
   :global {
     .ant-page-header-footer .ant-tabs .ant-tabs-tab {
-      font-size: $font-size-14;
```

Comment:
I think, it will be difficult to look up the particular font-size to know its size here. Also i'm not sure if we need to introduce manual font sizes. If we need to update For Eg: table column header size, i think its better to override antd variables https://ant.design/docs/react/customize-theme. What do you think is the use case here ?

Reply:
I guess in this way we can keep spacing, font sizes, colors consistent throughout the app.

Reply:
If we go with this approach of abstracting, I'd vote for to abstract the font-size variable into a $header-font-size in scss.